jobbergate-api/jobbergate_api/apps/job_submissions/constants.py

```diff
@@ -0,0 +1,7 @@
+from auto_name_enum import AutoNameEnum, NoMangleMixin, auto
+
+
+class JobSubmissionStatus(AutoNameEnum, NoMangleMixin):
+    CREATED = auto()
+    SUBMITTED = auto()
+    COMPLETED = auto()
```

Comment:
Can you help me understand what this extra components are used for?

Could this simple be

```python
class JobSubmissionStatus(AutoNameEnum, NoMangleMixin):
    CREATED = "CREATED"
    SUBMITTED = "SUBMITTED"
    COMPLETED = "COMPLETED"
```
?